Add swagger for dataWarehouseUserActivities API

YiWuti opened the pull request:
For dataWarehouseUserActivities API,
add swagger file
add example file
add a line in Readme

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

YiWuti committed:
Add swagger for dataWarehouseUserActivities

Add swagger for dataWarehouseUserActivities

YiWuti committed:
add example file

add example file

YiWuti committed:
correct case sensitive name

datawarehouseUserActivities to dataWarehouseUserActivities

AutorestCI commented:
This commit was treated and no generation was made for Azure/azure-sdk-for-python

AutorestCI commented:
This commit was treated and no generation was made for Azure/azure-sdk-for-go

fearthecowboy commented:
You have some validation errors: 
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/335869076

Can you take a quick look?

YiWuti committed:
add dataWarehouseUserActivityName

add dataWarehouseUserActivityName

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-go

jaredmoo commented:
Please do not merge until I have approved. Thx

YiWuti committed:
remove "value"

GET ".../activities/current" should be a get single API, so it should not have "value"

YiWuti commented:
Links broken:

The "see this page" link in "There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, see this page." 

and the " Swagger checklist" link in "The spec follows the guidelines described in the Swagger checklist document."

YiWuti committed:
Replace tab spaces with blank spaces

YiWuti committed:
add missing "body"

fearthecowboy commented:
Looks good to me? OK to merge?

AutorestCI commented:
# Automation for azure-sdk-for-node

YiWuti commented:
@fearthecowboy Where will the online doc be generated? Could you give me a link?

fearthecowboy commented:
uh, wherever docs are normally generated on docs.microsoft.com ... really, sorry, I'm not really sure where they go..
